pkgs/by-name/as/astro-language-server/package.nix

```diff
@@ -35,15 +35,20 @@ stdenv.mkDerivation (finalAttrs: {
 
   buildInputs = [ nodejs_22 ];
 
-  pnpmWorkspace = "@astrojs/language-server";
+  # Must specify to download dependencies for its dependencies (required for worspace with yaml2ts)
```

Comment:
I think this comment has a typo error (required for worspace -> required for workspace)

Reply:
absolutely... I'm going to fix ^^. Ty